/**:
  ros__parameters:

    # Смещения координат для определения целевых
    x_offset: 1.8
    y_offset: -1.9

    threshold: 0.5 # Порог по расстоянию между точками

    speed: 0.5     # Масимальная скорость движения по туннелю
    out_speed: 0.5 # Скорость после выезда из туннеля

    out_offset: 0.0 # Смещение после выезда с туннеля

    max_distance: 0.9 # Максимальная дистанция для начала объезда препятствий

    max_len: 50 # Размер буфера с угловыми скоростями

    # PI константы для скорости
    Kp: 0.42
    Ki: 0.003